tests/ProjectDeleteTest.php

```diff
@@ -71,37 +106,7 @@ public function testPurgeExpiredProjectRemoveJoinRequest()
         $this->assertCount(1, $joinRequests);
         $this->assertSame($project['id'], $joinRequests[0]['project']['id']);
 
-        $this->client->updateProject($project['id'], ['expirationDays' => -1]);
```

Comment:
Tenhle řádek bys měl buď tady nechat (máš to i na ostatních místech) + vyhodit ho z metody `waitUntilProjectWillBeDeleted`. Případně naopak ho nechat v té metodě a všude jinde odebrat.